src/java/org/apache/cassandra/metrics/CassandraMetricsRegistry.java

```diff
@@ -41,15 +57,20 @@
  */
 public class CassandraMetricsRegistry extends MetricRegistry
 {
-    public static final CassandraMetricsRegistry Metrics = new CassandraMetricsRegistry();
+    private static final Logger logger = LoggerFactory.getLogger(CassandraMetricsRegistry.class);
+
+    public static final CassandraMetricsRegistry Metrics = new CassandraMetricsRegistry(Ints.checkedCast(TimeUnit.MICROSECONDS.toNanos(1)));
```

Comment:
It's noted in the JIRA issue itself that it's enough to read the metrics once a day, but `MICROSECONDS` is here. Is it correct?

Reply:
Yeah it should be days to micros here.